src/main/java/com/dev/cinema/model/User.java

```diff
@@ -27,8 +29,6 @@
     @Column(name = "user_password")
     private String password;
 
-    public User(String email, String password) {
-        this.email = email;
-        this.password = password;
-    }
+    @ManyToMany
+    private Set<Role> userRoles;
```

Comment:
let's join fetch roles in findUserByEmail() method?

Reply:
done